planner/cascades/testdata/transformation_rules_suite_in.json

```diff
@@ -15,7 +15,8 @@
       "select a, f from (select a, f, g from t where f = 1) t1 where g > 1",
       "select a, f from t where g > 1 and f > 1",
       "select t1.a, t1.b from t t1, t t2 where t1.a = t2.a and t1.a = 10 and t2.a = 5",
-      "select a, b from ((select a, b from t) union all(select c as a, d as b from t)) as t1 where a > 1"
+      "select a, b from ((select a, b from t) union all(select c as a, d as b from t)) as t1 where a > 1",
+      "select a, b, row_number()over(partition by a order by b desc) as n from t"
```

Comment:
add `where a > 1` at the end of this test case, otherwise it won't trigger the new rule.